application/cache/cache.go

```diff
@@ -0,0 +1,233 @@
+package cache
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/go-redis/cache/v8"
+	"github.com/go-redis/redis/v8"
+
+	"github.com/silinternational/wecarry-api/api"
+	"github.com/silinternational/wecarry-api/conversions"
+	"github.com/silinternational/wecarry-api/models"
+)
+
+var (
+	RequestsCache         *cache.Cache
+	PublicRequestsKeyName string = "public"
+)
+
+func init() {
+	ring := redis.NewRing(&redis.RingOptions{
+		Addrs: map[string]string{
+			"redis": "redis:6379",
```

Comment:
These will need to be updated to obtain from the env. Can this package import `domain`? We load most of the env config there and then access them via `domain.WhateverConfig`

Reply:
Yes, I'll move the config to there